dashboard/src/actions/overviewActions.js

```diff
@@ -72,9 +72,11 @@ const initializeRuns = () => (dispatch, getState) => {
   });
 
   const savedRuns = data.filter(
-    (item) => item.metadata.global?.dashboard.saved
+    (item) => item.metadata.global?.dashboard?.saved
+  );
+  const newRuns = data.filter(
+    (item) => !item.metadata.global?.dashboard?.saved
```

Comment:
I didn't notice this before; but should `newRuns` be based on `!item.metadata.global?.dashboard?.seen` instead??? Or is "new" really supposed to be just "not saved"??

Reply:
New runs should be not saved ones :worried: